security/container_security/security-container-signature.adoc

```diff
@@ -0,0 +1,14 @@
+[id="security-container-signature"]
+= Container image signatures
+
+Red Hat delivers signatures for the images in the Red Hat Container Registries. Those signatures can be automatically verified when being pulled to OpenShift Container Platform (OCP) 4 clusters using Machine Config.
+
+Most of the images that comprise OCP 4 are shipped from quay.io, and only the release image is signed. Release images refer to the approved OCP 4 images, offering a degree of protection against supply chain attacks. However, some extensions to OCP 4 such as logging, monitoring, and service mesh are shipped as operators from the Operator Lifecycle Manager (OLM). Those images ship from the Red Hat Container Registry (registry.redhat.io).
```

Comment:
s/comprise/compose

Suggested edits for Quay.io:
s/quay.io/link:https://quay.io/[Quay.io]

or 

s/quay.io/Quay.io

s/operators/Operators

Note: we always capitalize Operators

s/Red Hat Container Registry (registry.redhat.io)/link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog Container images] registry

or

s/Red Hat Container Registry (registry.redhat.io)/lRed Hat Ecosystem Catalog Container images registry

Note: registry.redhat.io had a name change to Red Hat Ecosystem Catalog.